packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -1,11 +1,24 @@
 import logging
+from subprocess import check_output
+
+import pytest
 
 import requests
+
 import retrying
 
+from pkgpanda.build import load_json
 from test_util.marathon import get_test_app, get_test_app_in_docker
 
 
+def config():
```

Comment:
This function isnt used except for in lb_enabled, so just delete it and do load_json in lb_enabled